api-report/merge-tree.api.md

```diff
@@ -875,13 +876,16 @@ export class MergeBlock extends MergeNode implements IMergeBlock {
 
 // @public (undocumented)
 export class MergeNode implements IMergeNodeCommon {
+    set cachedLength(n: number | undefined);
     // (undocumented)
-    cachedLength: number;
+    get cachedLength(): number;
     // (undocumented)
     index: number;
     // (undocumented)
     isLeaf(): boolean;
     // (undocumented)
+    nullableCachedLength: number | undefined;
```

Comment:
chose to create a new field rather than update the old field to reduce code churn -- if we make `cachedLength` nullable, we have to add `?? 0` in a lot of places, which I don't like because `??` precedence can be a footgun and it updates hundreds of lines